src/main/java/org/traccar/model/Position.java

```diff
@@ -94,6 +95,10 @@
 
     public static final String KEY_DRIVER_UNIQUE_ID = "driverUniqueId";
 
+    public static final String KEY_CNG_STATUS = "cngStatus"; // in use
```

Comment:
New parameters are added to `Position` class only after very careful consideration and only if at very minimum 2 protocols use it.